pkgs/by-name/po/podman/package.nix

```diff
@@ -46,6 +47,7 @@ let
       util-linux
       iptables
       iproute2
+      nftables
```

Comment:
Shouldn't we add this via extraPackages in the podman module based on if nft for the firewall ist used?

Reply:
I think it is appropriate here because iptables is also added here